.circleci/continue_config.yml

```diff
@@ -447,6 +447,16 @@ workflows:
           enabled: true
           requires:
             - ci-base-<< matrix.python-version >>-<< matrix.extras >>
+      - integration-test:
```

Comment:
Let's revert this as soon as #683 merges

Reply:
Yup, created a ticket in linear to keep track: https://linear.app/kolena/issue/KOL-7444/use-small-resource-instance-for-client-repo-tests

Reply:
Whoops, wrong line.